src/platform/logging/index.ts

```diff
@@ -80,7 +80,7 @@ export function traceInfoIfCI(message: string, ...args: Arguments): void {
 /** Logging Decorators go here */
 
 export function traceDecoratorVerbose(message: string, opts: TraceOptions = DEFAULT_OPTS): TraceDecoratorType {
-    return createTracingDecorator({ message, opts, level: LogLevel.Debug });
+    return createTracingDecorator({ message, opts, level: LogLevel.Trace });
```

Comment:
Also found that verbose logging is used in a number of places, verbose is not the same as debug.
This terminology is confusing - carry on from Python extension ( i think we should just make it consistent instead of having verbose = debug & info = trace, & the like).
But thats a debt item